Sources/PackageDescription/Context.swift

```diff
@@ -22,7 +22,19 @@ public struct Context {
     public static var packageDirectory : String {
         model.packageDirectory
     }
-    
+
+    /// Information about the git status of a given package, if available.
+    @available(_PackageDescription, introduced: 5.11)
+    public static var gitInformation: GitInformation? {
```

Comment:
Just to bikeshed the name a bit: do we want to encode it as git specifically, or would a more vague "`sourceControl`" label be appropriate?

Reply:
Don't have a strong opinion, but I somewhat dislike that we're vague about this in other contexts.